assets/runner_cell/src/K8s.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Pool from "./Pool";
+
+export default function K8s({ fields, handleChange, handleBlur }) {
+  return (
+    <div>
+      <Pool
+        fi
```

Comment:
Let's move the `Pool` contents here, otherwise `K8s` is just a proxy component :D

Reply:
Oh ignore me, I see that's the part shared with Fly!